src/translation.ts

```diff
@@ -26,6 +29,19 @@ export function setTranslationMethod(tfn: TranslationFunction, loaded: () => boo
   _loaded = loaded;
 }
 
+/**
+ * If no translation function has been set, this will mark the translation are loaded.
+ *
+ * By default, the translations should not be set as loaded, otherwise top-level translated constants will never be
+ * translated. But if by the time the model is instantiated no custom translation function has been set, we can set
+ * the default translation function as loaded so o-spreadsheet can be run in standalone with no translations.
+ */
+export function setDefaultTranslationAsLoaded() {
```

Comment:
IMO translation loaded is something internal, I would rename it to `setDefaultTranslationMethod`. It's clearer from the caller.